root/cdtoc/attach_filter_release.tt

```diff
@@ -61,7 +63,7 @@
             [%- IF was_mbid_search;
                   l('We couldn’t find a release matching that MBID.');
                 ELSE;
-                  l('No results found. Try refining your search query.');
+                  l('No results found. For performance issues, this search checks only a limited amount of releases closest to your entered query. To ensure a better result, search for the full release title or as close to it as possible. You can also {search_link|search manually} and paste the link to the release here.', { search_link => { href => search_link, target => '_blank' } });
```

Comment:
For readability and to limit i18n breakage and redundancy, I suggest to keep the initial paragraph (for the short answer), and to add the new text in a separate paragraph (for the long answer).

About the added text itself now:

1. The first added sentence doesn’t really makes clear that the title is searched at first, and that other search conditions are tested on these limited amount of intermediate results.
2. The second added sentence is more about trying to work around the explained limitation than actually _ensuring a better result_.
3. The third and last added sentence provides a link for a search for release by name. It is not clearly worded. Would it be possible to pass some other conditions such as CD format and whatnot? See for example: https://musicbrainz.org/search?query=release%3A%28%22Never+Gonna+Give+You+Up%22%29+AND+format%3ACD&type=release&limit=25&method=advanced

Reply:
For 3. we could, but the system doesn't map entirely great right now, because a) we don't always know it's a CD (other formats also support discids, including no format set) and b) the main thing we could use to search (number of tracks) clashes because we want number of *cdtoc* tracks (without data tracks) but indexed search just has total number :)